framework/src/modules/interoperability/base_interoperability_endpoint.ts

```diff
@@ -148,6 +149,26 @@ export abstract class BaseInteroperabilityEndpoint<
 		};
 	}
 
+	public async getChainValidators(
+		context: ModuleEndpointContext,
+		chainID: Buffer,
+	): Promise<ChainValidators> {
+		const interoperabilityStore = this.getInteroperabilityStore(context);
+		const chainValidators = await interoperabilityStore.getChainValidators(chainID);
```

Comment:
Please use `ChainValidatorsStore` class instead of interoperabilityStore as we are removing it from store and will not be valid, so you will need to move the logic from store to here

Reply:
Ok then the issue was not written correctly as it says: `Implement getChainValidators in BaseInteroperabilityStore and call it in the endpoint`.

Reply:
@ishantiw @Phanco As per our discussion, we have decided to remove both functions from base store class in lieu of the upcoming PR: https://github.com/LiskHQ/lisk-sdk/pull/7700.  The functions will remain in base endpoint.

Furthermore, the tests for the endpoints were removed in this PR as they will be implemented in PR: https://github.com/LiskHQ/lisk-sdk/pull/7700